src/main/java/com/guciowons/shoppingbasket/Basket/BasketController.java

```diff
@@ -19,11 +20,19 @@ public ResponseEntity<Basket> createBasket(){
     }
 
     @PutMapping(value = "/{basketId}/{productId}/{quantity}")
-    public ResponseEntity<BasketSummarized> addProductToBasket(@PathVariable int basketId, @PathVariable int productId, @PathVariable int quantity){
+    public ResponseEntity addProductToBasket(@PathVariable int basketId, @PathVariable int productId, @PathVariable int quantity){
         try{
-            return new ResponseEntity<>(basketService.addProductToBasket(basketId, productId, quantity), HttpStatus.ACCEPTED);
+            return ResponseEntity
+                    .status(HttpStatus.ACCEPTED)
```

Comment:
Zobacz co znaczy dokładnie ACCEPTED.

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Jak najbardziej możesz to tak zwrócić, ale nie do końca jest to ten kod, który powinien być zwrócony w tej sytuacji. Bardziej odpowiednie będzie 200 - OK.